packages/components/src/__actions__/Menu/v3/MenuItem.module.scss

```diff
@@ -0,0 +1,41 @@
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/spacing";
+@import "~@kaizen/design-tokens/sass/typography";
+
+.item {
+  font-family: $typography-paragraph-body-font-family;
+  font-size: $typography-paragraph-body-font-size;
+  letter-spacing: $typography-paragraph-body-letter-spacing;
+  font-weight: $typography-paragraph-body-font-weight;
+  line-height: $typography-paragraph-body-line-height;
+  color: rgba($color-purple-800-rgb, 0.7);
+  padding: $spacing-6 $spacing-8;
+  border: $border-focus-ring-border-width $border-focus-ring-border-style
+    transparent;
+  border-radius: 4px;
+  display: flex;
+  align-items: center;
+  margin-inline: $spacing-xs;
```

Comment:
Can we update this to `$spacing-6` instead of the t-shirt size?

Reply:
Done :)

(should have covered all tshirt usages in this PR)